spotipy/cache_handler.py

```diff
@@ -107,3 +113,49 @@ def get_cached_token(self):
 
     def save_token_to_cache(self, token_info):
         self.token_info = token_info
+
+
+class RedisCacheHandler(CacheHandler):
+    """
+    A cache handler that stores the token info in the Redis.
+    """
+
+    def __init__(self, redis, key=None):
+        """
+        Parameters:
+            * redis: Redis object provided by redis-py library
+            (https://github.com/redis/redis-py)
+            * key: May be supplied, will otherwise be generated
+                   (takes precedence over `token_info`)
+        """
+        self.redis = redis
+        self.key = key if key else 'token_info'
+
+        try:
+            from redis import RedisError
+        except ImportError:
+            warnings.warn(
+                'Error importing module "redis", '
+                'required for RedisCacheHandler',
```

Comment:
This is a comma splice. I suggest changing it to `Error importing module "redis"; it is required for RedisCacheHandler`

Reply:
Done